src/components/source_list.js

```diff
@@ -120,4 +107,4 @@ SourceList.defaultProps = {
   withGrouping: false
```

Comment:
showSearch and searchRenderer are not used here. move the defaults to appropriate place